forge/ee/db/models/FlowTemplate.js

```diff
@@ -62,6 +62,15 @@ module.exports = {
             get () {
                 const rawValue = this.getDataValue('modules') || '{}'
                 return JSON.parse(rawValue)
+            },
+            validate: {
+                isValidModules (value) {
+                    try {
+                        JSON.parse(value)
+                    } catch (err) {
+                        throw new Error('Invalid \'modules\' json')
+                    }
+                }
```

Comment:
Could I suggest we do a little more validation - I have been caught out with the lack of validation on blueprints myself in the past!

for example `null`, `""`, `"123"`, 123, `[]` and `[ {"name: "node-red-dashboard", "version": "1.9.1"}]` are ALL valid JSON but they will break the blueprint.

```suggestion
            isValidModules (value) {
                // modules should be a single level key:string-value:string object
                try {
                    const parsed = JSON.parse(value)
                    if (!parsed || typeof parsed !== 'object' || Array.isArray(parsed)) {
                        throw new Error()
                    }
                    for (const key in parsed) {
                        if (typeof key !== 'string' || typeof parsed[key] !== 'string') {
                            throw new Error()
                        }
                    }
                } catch (err) {
                    throw new Error("Invalid 'modules' json")
                }
            }
```

Reply:
_If you are to accept my suggestion, please double check indentation - looks like I am 1 block off_ (sorry)